container/apptabs.go

```diff
@@ -237,10 +238,18 @@ func (t *AppTabs) setSelected(selected int) {
 	t.current = selected
 }
 
+func (t *AppTabs) setTransitioning(transitioning bool) {
+	t.isTransitioning = transitioning
+}
+
 func (t *AppTabs) tabLocation() TabLocation {
 	return t.location
 }
 
+func (t *AppTabs) transitioning() bool {
+	return t.isTransitioning
+}
+
```

Comment:
Do we really need functions for this if they just return the value? I know that it kind of already is like that in the file though.